incubator/twitter/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "kubeless-python-simple",
+  "version": "0.0.1",
+  "description": "This example demonstrates how to setup a simple Python function with Kubeless",
+  "dependencies": {
+    "serverless-kubeless": "^0.1.4"
```

Comment:
but the function loads the requirements from the requirements.txt file. This package.json only defines how to install the serverless-kubeless node module.

Yes, I tested manually deploying the function with kubeless and with the serverless plugin.